contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelItem.vue

```diff
@@ -202,14 +204,14 @@
     <!-- Delete dialog -->
     <KModal
       v-if="deleteDialog"
-      :title="$tr('deleteTitle')"
-      :submitText="$tr('deleteChannel')"
+      :title="canEdit ? $tr('deleteTitle') : $tr('removeTitle')"
+      :submitText="canEdit ? $tr('deleteChannel') : $tr('removeChannel')"
```

Comment:
The button label for view-only users should be "Remove" according to https://github.com/learningequality/studio/pull/4918#issuecomment-2807098025 . You are currently using the `removeChannel` string that translates to "Remove from channel list". Let's create a new string with the value "Remove" and use that instead;
```
removeBtnLabel: "Remove"
```